src/Microsoft.Sbom.Api/Entities/ErrorType.cs

```diff
@@ -38,6 +38,9 @@ public enum ErrorType
         JsonSerializationError = 8,
 
         [EnumMember(Value = "Unsupported hash algorithm")]
-        UnsupportedHashAlgorithm = 9
+        UnsupportedHashAlgorithm = 9,
+
+        [EnumMember(Value = "Referenced SBOM file")]
```

Comment:
Why is this an error?

Reply:
This is a referenced SBOM, which can sometimes be outside of the drop and not present in the drop. So, we skip referenced SBOMs from validation.